src/utils/index.js

```diff
@@ -1,13 +1,12 @@
 /**
  * Get the size for the string value.
+ * Returns 1 with an empty string.
  *
  * @param {Object} value - The value.
  *
  * @return {Number} The size.
  */
-const size = (value) => {
+export const fieldStringLen = (value) => {
   const length = String(value).length;
```

Comment:
Unrelated, but:

```suggestion
  const length = [...String(value)].length;
```

because `('🎉').length === 2` but `[...'🎉'].length === 1`?

Reply:
TIL
I think it makes sense to fix this, however we're using this length variable to update the width of the input fields. With pretty large characters like 🎉 it actually helps having their length as 2 to avoid the input field width from being too narrow for all of the characters, making having a scroll bar.
Probably something we want to improve at some point, variable width input fields where characters can be unicode make things tricky.

Reply:
Yeah, I see :) There’s also things like https://www.npmjs.com/package/string-width but I think they are more CLI-focused and probably would need some more specific config options